slides/slides.md

```diff
@@ -39,7 +39,7 @@ backgroundImage: url('https://marp.app/assets/hero-background.jpg')
 - Provide consistent spacing, colours, etc
 - The ability to make changes with confidence
 
-----
+---
```

Comment:
There was a mix of `----` and `---`.
Minimum necessary is `---` so I opted for the shorter.